resources/qml/PrintSetupSelector/Custom/MenuButton.qml

```diff
@@ -18,11 +18,11 @@ Button
     property string labelText: ""
     id: button
     hoverEnabled: true
-    leftPadding:UM.Theme.getSize("wide_margin").width
+    leftPadding: UM.Theme.getSize("default_margin").width
 
     background: Rectangle
     {
-        id: backgroundRectangle
+        id: backgroundRectanglewide_margin
```

Comment:
I think it's bad practice to mix snake case, camel case (and no case?) in the same id. As the id is never referred to I would remove this id entirely.
```suggestion
```